### DISCUSSION TITLE
Low and High alerts cancelled by Xdrip because of volume of 0.0

### DISCUSSION THREAD

**Discussion by andyrozman** [NightscoutFoundation/xDrip#2751] [Q&A] [answered]
Ever since I transfered my setting from my old phone to new one, I have problem with Alerts. First problem was that some sounds were not played, because I had wrong sound file configured (seems like copying config from one phone to another is not always sucessful, even if phones are from same manuafacturer (but on different Android versions)). I fixed this problem by changing alert tone to xDrip Default. 

But now I have second issue... It seems that first tree times when alert is raised it is silenced by xDrip because volume is set to 0.0. Problem is that I can't find where this setting is set.  

In log I see message from AlertPlayer: "Not playing file as requested volume is 0.0"

Is there a setting I have missed? I am using xDrip version Alpha from few weeks ago (when fix for OnePlus was introduced)... Can't tell you exact version, because I don't see any "About" option...

**Comment by tzachi-dar** [comment, marked as answer]
I think it depends on your volume profile. If it is ascending then it will start low and grow within a few minutes.
You can change that by settings->alarms and alerts->alarms settings->profile.

**Reply by andyrozman** [reply]
I found the problem... In "Alarm settings" there is setting called "Delay Ascending Volume" which was enabled... Very weird...

**Reply by Navid200** [reply]
I had to set that to enabled because there is no way it would be approved since when users get used to something, we don't wan to confuse them.

The Ascending volume always started the sound with a delay.  As a user, I hated that.  So, I added the option to set the delay to 0.  But, it couldn't be set to default.
And. that's what you are seeing.

I now want the trend to become more linear so that you can hear it even better.  And no one has even reviewed it yet:
https://github.com/NightscoutFoundation/xDrip/pull/2743

**Reply by andyrozman** [reply]
It this something new, because I never noticed that on my old phone? I was using stable version of xDrip there...

**Reply by Navid200** [reply]
The no delay option PR was opened in September of 2021 and was merged in December of 2021.
https://github.com/NightscoutFoundation/xDrip/pull/1834

Nothing has changed with respect to that setting since then.

**Comment by Navid200** [comment]
I just saved settings from my main xDrip, which I have set to use the Ascending volume profile and with the delay ascending volume setting disabled.

I then, imported the saved settings into an fresh install of xDrip with all settings left at default.
It indeed chose the Ascending volume profile and disabled the "Delay ascending volume" setting.

Is it possible that you only transferred the database to your new phone and not the settings as well?

**Reply by andyrozman** [reply]
I definetly transfered the settings, because I didn't need to configure anything afterwards... But I did switch Alerts Tone from System to Xdrip Default (in last 2 weeks I think).
